### PR TITLE
Barrier sync example

### PR DESCRIPTION
**Pull request by dns2utf8**
I wrote an example with a barrier.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -26,6 +26,7 @@ extern crate threadpool;
 
 * [rust-scoped-pool](http://github.com/reem/rust-scoped-pool)
 * [scoped-threadpool-rs](https://github.com/Kimundi/scoped-threadpool-rs)
+* [crossbeam](https://github.com/aturon/crossbeam)
```
Comment (frewsxcv) [inline README.md]: Does this get used somewhere?

File: README.md [hunk 2]
```diff
@@ -13,7 +13,7 @@ Add this to your `Cargo.toml`:
 
 ```toml
 [dependencies]
-threadpool = "1.0"
+threadpool = "1.2"
```
Comment (frewsxcv) [inline README.md]: This change isn't necessary; specifying `threadpool = 1.0` will still download the latest `1.x` version

Comment (dns2utf8) [inline README.md]: Oh, I did not know that. I will change it back.

File: README.md (hunk 1 above)
Comment (dns2utf8) [inline README.md]: No, I just added it to the list of crates with a thread pool.

**Comment by dns2utf8** [comment]
@frewsxcv thank you for your review.
Should I rebase it?

File: README.md (hunk 1 above)
Comment (frewsxcv) [inline README.md]: Ignore my earlier comment, I thought this was the "links" section of the markdown document.

**Comment by frewsxcv** [comment]
> Should I rebase it?

Yep, looks great! Thanks!

**Comment by dns2utf8** [comment]
Thanks for the review!

**Pull request merged**